packages/cli/src/util/rollback/request-rollback.ts

```diff
@@ -28,97 +27,54 @@ export default async function requestRollback({
   project: Project;
   timeout?: string;
 }): Promise<number> {
-  const { config, output } = client;
+  const { output } = client;
   const { contextName } = await getScope(client);
 
-  if (!isValidName(deployId)) {
-    output.error(
-      `The provided argument "${deployId}" is not a valid deployment or project`
-    );
-    return 1;
-  }
-
-  let deployment: Deployment;
-  let team: Team | undefined;
-
   try {
-    output.spinner(
-      `Fetching deployment "${deployId}" in ${chalk.bold(contextName)}…`
-    );
+    const deployment = await getDeploymentByIdOrURL({
+      client,
+      contextName,
+      deployId,
+    });
 
-    const [teamResult, deploymentResult] = await Promise.allSettled([
-      config.currentTeam ? getTeamById(client, config.currentTeam) : undefined,
-      getDeployment(client, contextName, deployId),
-    ]);
+    // create the rollback
+    await client.fetch<any>(
```

Comment:
If we're not reading the response, I think we can drop `any`:

```suggestion
    await client.fetch(
```